cypress/fixtures/successful_saving_article_response.json

```diff
@@ -0,0 +1,7 @@
+{
+  "status": 200,
+  "headers": {},
+  "body": {
+    "message": "all good"
```

Comment:
Is this a real response you will be getting from the back-end on a successful post request?

Reply:
Solved, we just removed the 'message' field completely since we are not using it for this request